internal/pkg/deploy/cloudformation/app.go

```diff
@@ -45,14 +53,12 @@ func (cf CloudFormation) DeployApp(template, stackName, changeSetName, cfExecuti
 	}
 
 	// If error returned from CreateStack is ErrCodeAlreadyExistsException move on to CreateChangeSet flow.
-	if err != nil {
-		awsErr, ok := err.(awserr.Error)
-		if !ok {
-			return fmt.Errorf("create stack: %w", err)
-		}
-		if awsErr.Code() != cloudformation.ErrCodeAlreadyExistsException {
-			return fmt.Errorf("create stack: %w", err)
-		}
+	awsErr, ok := err.(awserr.Error)
```

Comment:
are you sure it's safe to remove the "err != nil"?

Reply:
my ide reminds me to do that and it is redundant because we have `if err == nil` and return anyway inside of the `if` clause.